src/d3a/d3a_core/d3a_jobs.py

```diff
@@ -72,11 +72,16 @@ def start(scenario, settings):
         config.area = scenario
 
     kwargs = {"no_export": True, "pricing_scheme": 0}
-    run_simulation(setup_module_name=scenario_name,
-                   simulation_config=config,
-                   slowdown=settings.get('slowdown', 0),
-                   redis_job_id=job.id,
-                   kwargs=kwargs)
+    try:
+        run_simulation(setup_module_name=scenario_name,
+                       simulation_config=config,
+                       slowdown=settings.get('slowdown', 0),
+                       redis_job_id=job.id,
+                       kwargs=kwargs)
+    except Exception:
+        import traceback
```

Comment:
Is there a reason for this local import? (also next line)

Reply:
I decided to add the imports here because they are really only needed in case the simulation crashes. By lazily importing these here we ensure that these modules are going to be imported only when the simulation crashes, and only in production mode. Therefore a CLI user will never import these modules.

Reply:
Makes sense! Thanks!